### PR TITLE
gave the target for Bender

### PR DESCRIPTION
**Pull request by Andrii-Kutil**

### CODE REVIEW

File: src/main/java/core/basesyntax/FieldXY.java [hunk 1]
```diff
@@ -12,8 +12,40 @@
  */
 public class FieldXY {
 
+    public void gogoBender(Robot robot, long i) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FieldXY.java:15]: it's funny) but let's change it to more informative name)

File: src/main/java/core/basesyntax/FieldXY.java [hunk 2]
```diff
@@ -12,8 +12,40 @@
...
+
+            gogoBender(robot, (long) robot.getY() - (long) toY);
+
+        } else if (robot.getY() < toY) {
+            while (robot.getDirection() != Direction.UP) {
+                robot.turnRight();
+            }
+            gogoBender(robot, (long) toY - (long) robot.getY());
+        }
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FieldXY.java:49]: Try to move repeating logic to method.

Comment (Andrii-Kutil) [reply src/main/java/core/basesyntax/FieldXY.java]: Done, I created the new method.

File: src/main/java/core/basesyntax/FieldXY.java (hunk 1 above)
Comment (Andrii-Kutil) [reply src/main/java/core/basesyntax/FieldXY.java]: Noooo, I has fixed)

**Pull request closed without merging**